pixl_ehr/src/pixl_ehr/main.py

```diff
@@ -58,6 +63,21 @@ async def startup_event() -> None:
         task.add_done_callback(background_tasks.discard)
 
 
+@app.post(
+    "/export-radiology-as-parquet",
+    summary="Copy all radiology reports in the PIXL DB to a parquet file",
+)
+def export_radiology_as_parquet(project_name: str, extract_datetime: datetime) -> None:
+    """Batch export of all radiology reports in PIXL DB to a parquet file."""
+    # This command has to tell us the extract identifiers so we can extract to the correct
+    # path, however we can't verify that the reports in the DB actually came from that extract.
+    # Also we can't check that all reports in the queue have been processed.
+    # So we are relying on the user passing correct parameters here.
```

Comment:
I _think_ that's what we landed on doing so the comment may be out of date, or I've misunderstood

Reply:
I don't know what we agreed in the end. Should they not go in the `extract` table since they are properties of an extract? But then we don't link the `ehr_*` tables to `image` or `extract` (perhaps because the images might not exist yet - but the extracts will!) so it could be hard.
Relatedly, can we move the contents of `pixl-db_init.sh` to `create_pixl_tbls.py` - would be nice to have the DB schema all in one place.

Reply:
> I don't know what we agreed in the end. Should they not go in the extract table since they are properties of an extract? But then we don't link the ehr_* tables to image or extract (perhaps because the images might not exist yet - but the extracts will!) so it could be hard.

I think for now, we should pretend like the `extract` table doesn't exist so that we can control it independently, a later stage would be rationalisting (and normalising) it in sqlaclhemy later on. We should make sure we store each `ehr_` row with the `project_name`  and `extract_datetime`.

>  Relatedly, can we move the contents of pixl-db_init.sh to create_pixl_tbls.py - would be nice to have the DB schema all in one place.

Yes though, I'd say that this is a later task if we get everything else that we need done and have extra time.